config/clusters/awi-ciroh/prod.values.yaml

```
@@ -49,6 +49,36 @@ basehub:
                   slug: "new"
                   kubespawner_override:
                     image: "quay.io/2i2c/awi-ciroh-image:79bb68961852"
+                devcon1:
+                  display_name: CNN for downscaling climate forcing variables
+                  default: true
+                  slug: "devcon1"
+                  kubespawner_override:
+                    image: "quay.io/benlee7411/devcon24:swolvin"
```

```suggestion
                    image: "quay.io/benlee7411/devcon24:swolvin"
                    image_pull_policy: "Always"
```

I think you are planning on simply updating these tags as needed, is that right? If so, let's add this `image_pull_policy` to all of these as well, so that image tag caching is disabled.

---

Added